acsoo/templates/project/+project.name+/deploy-test.bob

```diff
@@ -1,8 +1,13 @@
-#!/usr/bin/env python
+{{% if python_version == 'python3' %}}
+{{% set config_parser_lib = 'configparser' %}}
+{{% else %}}
+{{% set config_parser_lib = 'ConfigParser' %}}
+{{% endif %}}
+#!/usr/bin/env {{{ python_version }}}
```

Comment:
Is this not inserting a blank line before `#!/usr/bin/env`?

Reply:
There isn't any blank line after generating the project.